buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -279,7 +253,8 @@ RUN cd /tmp \
   && cd .. \
   && rm -rf openssl-OpenSSL_1_1_1b \
   && echo "/usr/local/openssl-1.1.1b/lib" > /etc/ld.so.conf.d/openssl-1.1.1b.conf \
```

Comment:
This doesn't become effective until `ldconfig` is executed, but that step is missing.

Reply:
`ldconfig` is run at the [end of the protobuf step](https://github.com/eclipse/openj9/pull/6054/files#diff-98f2a595424a7af76747ea2c636f6d19R274). Is this sufficient, or should it be run again at the end of the Openssl step?

Reply:
I think running ldconfig once is fine, but I don't think it should be tied to a specific section if it's required for more than 1 section. Ie. We should move it to it's own step and leave a comment that at least these 2 packages rely on it.

Reply:
I think a comment in this section about `ldconfig` that will be run later is appropriate.